ui/datasets/templates/semantic-ui/datasets/DepositEdit.jinja

```diff
@@ -0,0 +1,25 @@
+{#def
+    theme,
+    forms_config,
+    searchbar_config,
+    record,
+    extra_context,
+    ui_links,
+    permissions
+#}
+
+{% extends "oarepo_ui/deposit.html" %}
+
+
+{%- block page_title %}
+    {{ _('Deposit - Metadata Record') }}
```

Comment:
Edit Metadata Record :)

Reply:
Word deposit or draft shouldn't be shown anywhere to user (the record is metadata-only, already exists, is published & only possible action is to edit it)